src/Compilers/CSharp/Portable/Binder/Binder_Lookup.cs

```diff
@@ -1474,7 +1480,7 @@ private static bool IsInstance(Symbol symbol)
         // Check if the given symbol can be accessed with the given arity. If OK, return false.
         // If not OK, return true and return a diagnosticinfo. Note that methods with type arguments
         // can be accesses with arity zero due to type inference (but non types).
-        private static bool WrongArity(Symbol symbol, int arity, bool diagnose, LookupOptions options, out DiagnosticInfo diagInfo)
+        private static bool WrongArity(Symbol symbol, int arity, bool diagnose, LookupOptions options, CSharpCompilation compilation, out DiagnosticInfo diagInfo)
```

Comment:
We'll want to test `[GenericAttribute(1)]` where `GenericAttribute<T>`.
What is the desired language behavior (infer type arguments or not)? Depending on that choice, the logic below may be wrong.

Reply:
Now it not inferring the arguments
I think it's like calling to constructor so no need to infer